dockstore-common/src/main/java/io/dockstore/common/yaml/BriefToStringBuilder.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dockstore.common.yaml;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+public class BriefToStringBuilder extends ToStringBuilder {
```

Comment:
Is this likely to be used anywhere but `YamlAuthor`?
If it isn't, it might be more clear if this was an inner class in that class.

Reply:
Eventually, we should use `BriefToStringBuilder` to define similar `toString`s for all of the yaml objects (like `YamlWorkflow`), so when those objects are represented in error messages, we're more likely to display useful information to the user (rather than the default `toString` representation, which never helps).  Probably good to keep it a standalone class...

Reply:
Makes sense (and capture this as Javadoc)